### PR TITLE
Clean up and update dependencies

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Motivate by the need to update loader-utils for security.

## Code changes

Clean and update some dependencies (mainly build tool dependencies).

## User-facing changes

N/A

## Backwards-incompatible changes

N/A

### CODE REVIEW

File: jupyterlab/staging/package.json [hunk 1]
```diff
@@ -186,13 +186,13 @@
     "chokidar": "^3.4.0",
     "css-loader": "^6.7.1",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
-    "fs-extra": "^9.0.1",
+    "fs-extra": "^10.1.0",
```
Comment (jtpio) [inline jupyterlab/staging/package.json:189]: Let's discard these changes to staging as suggested by the CI check?

https://github.com/jupyterlab/jupyterlab/actions/runs/3487285189/jobs/5834734413

**Comment by jtpio** [comment]
`testutils` failure looks related: https://github.com/jupyterlab/jupyterlab/actions/runs/3489494908/jobs/5844051209

**Comment by fcollonval** [comment]
> `testutils` failure looks related: https://github.com/jupyterlab/jupyterlab/actions/runs/3489494908/jobs/5844051209

The test passed in the last commit. But I don't think it is related to this PR; I have seen it on other PRs today.

**Comment by jtpio** [comment]
ah ok, this one looked new.

We should look into what's happening there to avoid accumulating too many CI failures (https://github.com/jupyterlab/jupyterlab/issues/13438).

**Pull request merged**